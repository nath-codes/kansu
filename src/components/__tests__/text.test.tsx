import React from 'react'

import Text from '../text'
import { renderWithTheme } from './helpers'

describe('Text', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(<Text>foo</Text>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
