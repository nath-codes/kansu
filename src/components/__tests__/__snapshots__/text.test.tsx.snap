// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
<div
  className="sc-bdVaJa iGaYgC"
  style={
    Object {
      "opacity": 0,
      "transform": "translateY(1.5rem)",
    }
  }
>
  foo
</div>
`;
